doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -436,3 +435,32 @@ possible option.
 
    * :ref:`sphx_glr_auto_examples_connectivity_plot_probabilistic_atlas_extraction.py`
 
+Extraction of signals from seeds:\  :class:`NiftiSpheresMasker`.
+==================================================================
+
+The purpose of :class:`NiftiSpheresMasker` is to compute signals from
+seeds containing voxels in spheres. It makes it easy to get these signals once
+you have a list of coordinates.
+
+Seeds definition
+------------------
+
+A single seed is a sphere defined by the radius (in millimeters) and the
+coordinates (typically MNI or TAL) of its center.
+
+A set of seeds is thus defined by a list of 3D coordinates and a radius.
```

Comment:
Can you give an example here: one line of code that shows the typical input.

The reason being that for a user beginner in Python, it may not be clear what you are talking about.